### PR TITLE
#77 Discontinued old plugins

### LINKED ISSUE #77: Remove unused plugins
The pytest plugins in this repository are only used by other repos of the Exasol Integration Team.

### Tasks 
* [ ] check which plugins are no longer used
* [ ] Remove these plugins: directory and GitHub workflows

### LINKED ISSUE #79: Bug: pytest-backend doesn't work with Python 3.9
Need to make sure all modules include
`from __future__ import annotations`

### PR DESCRIPTION
**Pull request by ahsimb**
closes #77 
closes #79

### CODE REVIEW

File: pytest-itde/README.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
 pytest-exasol-itde Plugin
 =========================
+This project has been discontinued. For integration tests running on SaaS please use the `pytest-backend` plugin.
```
Comment (tkilias) [inline pytest-itde/README.rst:3]: Please, use the full name exasol-pytest-backend and link to the pypi page of it

File: pytest-saas/README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# pytest-exasol-saas Plugin
+pytest-exasol-saas Plugin
+=========================
+This project has been discontinued. For integration tests running on SaaS please use the `pytest-backend` plugin.
```
Comment (tkilias) [inline pytest-saas/README.md:3]: Same as https://github.com/exasol/pytest-plugins/pull/80/files#r1850785155

File: pytest-itde/README.rst (hunk 1 above)
Comment (ahsimb) [reply pytest-itde/README.rst]: OK, also it should be ITDE, not SaaS.

File: pytest-backend/doc/changes/changes_0.3.2.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# 0.3.2 - 2024-11-20
```
Comment (tkilias) [inline pytest-backend/doc/changes/changes_0.3.2.md:1]:
```suggestion
# 0.3.2 - 2024-11-21
```

File: pytest-itde/doc/changes/changes_0.2.1.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# 0.2.1 - 2024-11-20
```
Comment (tkilias) [inline pytest-itde/doc/changes/changes_0.2.1.md:1]:
```suggestion
# 0.2.1 - 2024-11-21
```

File: pytest-saas/doc/changes/changes_0.2.3.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# 0.2.3 - 2024-11-20
```
Comment (tkilias) [inline pytest-saas/doc/changes/changes_0.2.3.md:1]:
```suggestion
# 0.2.3 - 2024-11-21
```

**Review by tkilias — APPROVED** [review]

**Pull request merged**